oremda/display/matplotlib.py

```diff
@@ -0,0 +1,49 @@
+from typing import Dict
+
+from oremda.display import DisplayHandle
+from oremda.typing import DisplayType, IdType, Port
+
+import matplotlib.pyplot as plt
+
+
+class MatplotlibDisplayHandle(DisplayHandle):
```

Comment:
The reason why I didn't originally included any implementation of the display handle in oremda is that we would have to add matplotlib as a dependency of the library

Reply:
Right, I moved this into a separate file so at least we don't have a hard dependence on it.